src/firebase/FirestoreService.ts

```diff
@@ -12,5 +12,7 @@ const firebaseConfig = {
 
 export function initializeDatabase() {
     firebase.initializeApp(firebaseConfig);
-    return firebase.firestore();
 }
+
+
+// https://firebase.google.com/docs/firestore/manage-data/add-data
```

Comment:
🤔❓

Reply:
ah thats just a note to self of where to get further info from to continue adding methods

Reply:
I can remove if that's preferred

Reply:
We should avoid keeping notes to self in the code. :-)

Reply:
sure : )